Buildings/package.mo

```diff
@@ -576,8 +576,10 @@ its class name ends with the string <code>Beta</code>.
     </tr>
     <tr><td valign=\"top\">Buildings.Controls.OBC.CDL.Logical.Timer
         </td>
-        <td valign=\"top\">Updated implementation to output accumulated time when input is <code>true</code>.<br/>
-                         This is for <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/1212\">issue 1212</a>.
+        <td valign=\"top\">Updated implementation to output cumulative time during which the input was <code>true</code>.<br/>
```

Comment:
@JayHuLBL : You added this to the release notes of 6.0.0, but this needs to be in 7.0.0

Reply:
@mwetter 
The note was in the notes of 6.0.0, as the timer that can accumulate true time is in version 6.0.0. 
I will revert the change of the release notes of 6.0.0 but add new notes in 7.0.0 to record the changes made this time.